### ISSUE TITLE
Can we extract some ideas from firebase client tool?

### ISSUE THREAD

**Issue by giancorzo** [tektonlabs/genesis-generator#14]
Check out 
https://www.npmjs.com/package/firebase-tools

I think we can get a lot of ideas from this command line. To start with I like the idea of doing "list" to list all available templates.

**Comment by jopereyral** [comment]
I was looking also for the [vue-cli ](https://github.com/vuejs/vue-cli) project to get some ideas

**Comment by jopereyral** [comment]
Also this -> https://github.com/facebook/create-react-app

**Comment by jjercTK** [comment]
Hi, I was thinking on something like [this](https://www.filepicker.io/api/file/CbXdfJw0S2ycKjiSfRD2)

**Comment by jopereyral** [comment]
> Hi, I was thinking on something like [this](https://www.filepicker.io/api/file/CbXdfJw0S2ycKjiSfRD2)

Yep for me works, there are things that are related to stack tech and other nice things we can add optionally according to project

**Comment by giancorzo** [comment]
> > Hi, I was thinking on something like [this](https://www.filepicker.io/api/file/CbXdfJw0S2ycKjiSfRD2)
> 
> Yep for me works, there are things that are related to stack tech and other nice things we can add optionally according to project, I think our template structure (aka #11 ) must support that

I like it, like joca said this need to be part of the template config file that need to support specific question for each tech stack
